test/projectvessel.Tests/Tests.fs

```diff
@@ -41,3 +42,16 @@ let ``That applying the inverse of counter event yields the initial state`` () =
         actual = initialState
 
     Check.QuickThrowOnFailure prop
+
+[<Fact>]
+let ``That reading the testing csv file yields a csv with one data row`` () =
+    let csv = csvloader.Load("../../../../../data/i18n_test.csv")
+
+    let testi18n = csv.Rows |> Seq.map(fun row -> row.Key, row.Value) |> Map.ofSeq
+    Assert.Equal("testcontent", testi18n.["testname"])
+ 
+    for row in csv.Filter(fun row -> row.Key = "testname").Rows do
```

Comment:
`for...in` is one of the non-functional parts in F#, I am not sure if we can use this

Reply:
Aye will fix

Reply:
Just saw this is just a test, so I will merge this now 👍